inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/agent/AgentClusterNodeDTO.java

```diff
@@ -42,16 +50,57 @@
 @ApiModel("Agent cluster node info")
 public class AgentClusterNodeDTO {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(AgentClusterNodeDTO.class);
+
     @ApiModelProperty(value = "Agent group name")
     private String agentGroup;
 
+    @ApiModelProperty(value = "Module id list")
+    private List<Integer> moduleIdList;
+
+    @ApiModelProperty(value = "Agent restart time")
+    private Integer agentRestartTime = 0;
+
+    @ApiModelProperty(value = "Install restart time")
+    private Integer installRestartTime = 0;
+
+    @ApiModelProperty("History list of module")
+    private List<ModuleHistory> moduleHistoryList;
+
     /**
      * Get the dto instance from the request
      */
     public static AgentClusterNodeDTO getFromRequest(AgentClusterNodeRequest request, String extParams) {
-        AgentClusterNodeDTO dto = StringUtils.isNotBlank(extParams)
-                ? AgentClusterNodeDTO.getFromJson(extParams)
-                : new AgentClusterNodeDTO();
+        AgentClusterNodeDTO dto;
+        if (StringUtils.isNotBlank(extParams)) {
+            dto = AgentClusterNodeDTO.getFromJson(extParams);
```

Comment:
Ensure all variables are not null in  `AgentClusterNodeDTO.getFromJson `

Reply:
Done.